containers/datalab/content/run-debug.sh

```diff
@@ -17,7 +17,13 @@
 
 mkdir -p /content/datalab/notebooks
 mkdir -p /content/datalab/docs
-rsync -r /datalab/docs/* /content/datalab/docs
+
+if [ -d /content/datalab/docs/.git ]
+then
+  (cd /content/datalab/docs; git fetch --all; git reset --hard origin/master)
```

Comment:
Should it be "cd /content/datalab/docs/notebooks"? "Git clone" creates the notebooks folder and you have to be in that folder to run git fetch?